src/script/lua_api/l_mainmenu.cpp

```diff
@@ -577,6 +577,13 @@ int ModApiMainMenu::l_get_favorites(lua_State *L)
 			lua_settable(L, top_lvl2);
 		}
 
+		float ping = servers[i]["ping"].asFloat();
+		if (ping) {
```

Comment:
this `if` seems incorrect to me
what does `asFloat()` return if the element doesn't exist or isn't a float?